consensus/safety-rules/src/error.rs

```diff
@@ -51,6 +51,8 @@ pub enum Error {
     InconsistentExecutionResult(String, String),
     #[error("Invalid Ordered LedgerInfoWithSignatures: Empty or at least one of executed_state_id, version, or epoch_state are not dummy value: {0}")]
     InvalidOrderedLedgerInfo(String),
+    #[error("Waypoint is not up-to-date: Previous waypoint version {0}, current waypoint version {1}, provided epoch {2}, current epoch {3}")]
```

Comment:
Super nitty, but can we update this to error to be: `WaypointOutOfDate` and change the string:

"Waypoint out of date: Previous waypoint version {0}, updated version {1}, current epoch {2}, provided epoch {3}"

Reply:
Updated the error and string as suggested.